### PR TITLE
logs exception handling and common step definition

### PR DESCRIPTION
**Pull request by sawantpritam**
@itsankit-google @rmstar Please review.

### CODE REVIEW

File: src/main/java/io/cdap/e2e/utils/SeleniumHelper.java [hunk 1]
```diff
@@ -92,6 +92,11 @@ public static void dragAndDrop(WebElement from, WebElement to) {
     act.dragAndDrop(from, to).build().perform();
   }
 
+  public static void dragAndDropByOffset(WebElement element, int xOffset, int yOffset) {
```
Comment (itsankit-google) [inline src/main/java/io/cdap/e2e/utils/SeleniumHelper.java:98]: where is this method used? Can we have a javadoc for it?

File: src/main/java/stepsdesign/PipelineSteps.java [hunk 2]
```diff
@@ -190,6 +217,20 @@ public void runThePipelineInRuntime() throws InterruptedException {
     CdfPipelineRunAction.runClick();
   }
 
+  @Then("Run the Pipeline in Runtime with runtime arguments")
+  public void runThePipelineInRuntimeWithRuntimeArguments() throws InterruptedException {
+    CdfPipelineRunAction.runClick();
+    for (Map.Entry<String, String> entry : runtimeArguments.entrySet()) {
+      if (SeleniumHelper.verifyElementPresent(
+        CdfStudioLocators.runtimeArgsValueLocator.replace("RUNTIME_ARGS_KEY", entry.getKey()))) {
```
Comment (itsankit-google) [inline src/main/java/stepsdesign/PipelineSteps.java:225]: Iirc `runtime argument key` is automatically set after pipeline deployment, we need to only provide values. Why are we replacing keys as well?

Comment (sawantpritam) [reply src/main/java/stepsdesign/PipelineSteps.java:225]: We are just checking if that `runtimetime argument key` is displayed and entering value in the text area next to that key.

File: src/main/java/io/cdap/e2e/utils/SeleniumHelper.java (hunk 1 above)
Comment (sawantpritam) [reply src/main/java/io/cdap/e2e/utils/SeleniumHelper.java:98]: javadoc added.
When we have multiple sinks they gets overlapped on canvas so sometimes connection is established with wrong sink during automation. This method is used to align sinks properly on canvas to establish connection.

File: src/main/java/stepsdesign/PipelineSteps.java (hunk 2 above)
Comment (itsankit-google) [reply src/main/java/stepsdesign/PipelineSteps.java:225]: sg, thanks

File: src/main/java/io/cdap/e2e/utils/SeleniumHelper.java (hunk 1 above)
Comment (itsankit-google) [reply src/main/java/io/cdap/e2e/utils/SeleniumHelper.java:98]: got it, thanks

**Review by itsankit-google — APPROVED** [review]
LGTM

**Pull request merged**